docs/source/api_reference/models.rst

```diff
@@ -459,3 +462,9 @@ AttentiveFPModel
 
 .. autoclass:: deepchem.models.AttentiveFPModel
   :members:
+
+MPNNModel
+---------
+
+.. autoclass:: deepchem.models.MPNNModel
```

Comment:
The path is not correct. I seem the correct one is `deepchem.models.torch_models.MPNNModel`.
And, can you add some notes for importing this class? I think users will misunderstand they can import this class from `deepchem.models`.

Reply:
Where should I add notes for importing this class? Just here?

Reply:
> Just here?

Yes!

Reply:
Done.